src/sidebar/components/selection-tabs.js

```diff
@@ -13,21 +13,23 @@ import SvgIcon from './svg-icon';
 /**
  *  Display name of the tab and annotation count.
  */
-function Tab({
-  children,
-  count,
-  isWaitingToAnchor,
-  onChangeTab,
-  selected,
-  type,
-}) {
+function Tab({ children, count, isWaitingToAnchor, onSelect, selected }) {
+  const selectTab = () => {
+    if (!selected) {
+      onSelect();
+    }
```

Comment:
This check avoids selecting a tab twice if both the `onMouseDown` and `onClick` handlers below are triggered. A small change in observable behavior here is that clicking the already-selected tab used to clear the selection. That no longer happens, which I think makes sense.